downloader.go

```
@@ -63,25 +80,33 @@ func main() {
 	}
 
 	// Validate and expand config
-	if err := expandConfig(&conf); err != nil {
+	if err := conf.expand(*staging, *overrideLatest || *checkLatest); err != nil {
 		log.Fatal(err)
 	}
 
+	// Print new versions and exit
+	if *checkLatest {
+		_ = conf.printUpdates(os.Stdout)
```

we should not ignore the error, at least print it, it could be helpful to debug

---

The day printing to stdout fails on my mac will be the day I finally throw it out of the window.

We could capture the error here, but where should we print it too? If priting to stdout errors, doesn't seem to be any reasonable place to log the error to.

---

I converted it to simply use `fmt.Printf`. Sample output looks like this now:
```
2021/06/30 15:26:30 nrjmx 1.5.3 -> 1.6.2
2021/06/30 15:26:30 Fetching releases for nri-discovery-kubernetes...
  - name: nri-jmx
    version: 2.4.10
  - name: nrjmx
    version: 1.6.2
```